util/json.h

```diff
@@ -78,5 +78,6 @@ void json_object_add_byte_array(struct json_object *o, const char *k, unsigned c
 void json_object_add_nprix64(struct json_object *o, const char *k, uint64_t v);
 void json_object_add_uint_0nx(struct json_object *o, const char *k, __u32 v, int width);
 void json_object_add_0nprix64(struct json_object *o, const char *k, uint64_t v, int width);
+void json_object_add_format(struct json_object *o, const char *k, const char *format, ...);
```

Comment:
I suggest to stick to the above pattern and use the name as final postfix, so in this case just `string`. There is also already one helper which is called `.._value_string`. So this would make more sense to me.

`json_object_add_string`

Reply:
Fixed as suggested. Thank you.